docs/source/cookbook/aggregated_node.rst

```diff
@@ -0,0 +1,89 @@
+Aggregated nodes
```

Comment:
This documentation doesn't seem to be related to the code change. Could it be split into another MR?